### ISSUE TITLE
User Code CC support

### ISSUE THREAD

**Issue by AlCalzone** [zwave-js/backlog#25]
Controller MUST have a UI allowing the end user to see the following properties:
• Number of supported User Codes
• The list of last known set User Codes
• The current keypad mode, if the supporting node supports more than one (v2)
• The current set Admin Code, if the supporting node supports a Admin Code (v2) 

Note that Admin Code was renamed from Master Code in the latest spec revision. Z-Wave JS hasn't renamed it yet.

**Comment by marcelveldt** [comment]
This requires an architecture discussion

**Comment by raman325** [comment]
Yeah I'm going to create one eventually, I've just been focused on the low hanging fruit first

**Comment by raman325** [comment]
Started an architecture discussion on this topic: https://github.com/home-assistant/architecture/discussions/1000

**Comment by AlCalzone** [comment]
For now, we can disable support with https://github.com/zwave-js/zwave-js/pull/7821
